cli/tests/041_info_flag_location.out

```diff
@@ -0,0 +1,6 @@
+DENO_DIR location: "[WILDCARD]"
+Remote modules cache: "[WILDCARD]deps"
+Emitted modules cache: "[WILDCARD]gen"
+Language server registries cache: "[WILDCARD]registries"
```

Comment:
The location variant shouldn't include the global fields, for modularity's sake.
```suggestion
```

Reply:
Done

Reply:
I disagree with this. I don't understand why this would be more modular than including the other fields. Just seems like you need to run `deno info` multiple times if you want all info, which seems annoying.

Reply:
It's the same reason `deno info mod.ts` doesn't include the global fields. You'd run `deno info --location https://foo/` because you're interested in the local storage file for some specific origin, it's unlikely you'd want the global fields at the same time. `--location` shouldn't be considered a passive modifier of `deno info` since it would keep the other fields the same and just add new more specific ones. That makes `deno info --location <some_location>` more comparable to `deno info <some_module>` and it should be similarly disjoint.